commands/command_fetch.go

```diff
@@ -76,7 +78,7 @@ func fetchCommand(cmd *cobra.Command, args []string) {
 		if len(cfg.FetchIncludePaths()) > 0 || len(cfg.FetchExcludePaths()) > 0 {
 			Print("Ignoring global include / exclude paths to fulfil --all")
 		}
-		success = fetchAll()
+		success = fetchAll(excludeOIDs)
```

Comment:
A similar change would need to be made below this as well for `fetchRef()` and `fetchRecent()` to ensure this applies during all usages of a fetch.